webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -0,0 +1,41 @@
+import Jed from 'jed';
+import { deprecateObjectProperty } from '../../foreman_tools';
+
+class I18n {
+  constructor(locales) {
+    this.cheveronPrefix = window.I18N_MARK ? '\u00BB' : '';
+    this.cheveronSuffix = window.I18N_MARK ? '\u00AB' : '';
+    this.translate = this.translate.bind(this);
+    this.ngettext = this.ngettext.bind(this);
+    this.initLocale();
+    this.jed = new Jed(locales[this.locale]);
+  }
+  initLocale() {
+    const locale = document.getElementsByTagName('html')[0].lang;
+    this.locale = locale.length === 0 ? 'en' : locale.replace(/-/g, '_');
+  }
+  translate(...args) {
+    return `${this.cheveronPrefix}${this.jed.gettext(...args)}${this.cheveronSuffix}`;
+  }
+  ngettext(...args) {
+    return `${this.cheveronPrefix}${i18n.ngettext(...args)}${this.cheveronSuffix}`;
```

Comment:
Shouldn't it be `this.jed.ngettext` here? Just for consistency with `translate` above.